python/cugraph/tests/dask/mg_context.py

```diff
@@ -54,11 +54,12 @@ class MGContext:
         Number of devices to use, verification must be done prior to call
         to ensure that there are enough devices available.
```

Comment:
Update documentation to reflect all parameters (new and the rmm_managed_memory that was missing before)

Reply:
Thanks, fixed in the latest commit.